derive/tests/macros.rs

```diff
@@ -130,17 +142,28 @@ fn should_use_method_name_aliases() {
 
 	// then
 	let result1: Response = serde_json::from_str(&res1.unwrap()).unwrap();
-	assert_eq!(result1, serde_json::from_str(r#"{
+	assert_eq!(
+		result1,
+		serde_json::from_str(
+			r#"{
```

Comment:
That looks a bit ugly :P

Reply:
Do you want me to manually format this? It kind of defeats the point of using rustfmt and will cause errors in the ci.

Reply:
No, not at all. Just a nitpick, that rustfmt is not perfect, perhaps we can tune some options in the future to get rid of that.
In the other comment I meant the errors that causes the CI to fail. If CI is green I'm good to merge this.

Reply:
The error is a formatting issue on stable where it expects a semicolon but on beta/nightly it doesn't. rustfmt 1.0 (with default settings) is supposed to be formatting compatible between versions unless it's a bug. It is a bit unfortunate, but not really sure what to do.

Reply:
Let's run `rustfmt` only on stable then. We've been through some issues like this in the past in some other repo afair (after all we killed rustfmt completely).